libdevcore/Log.cpp

```diff
@@ -28,61 +28,55 @@
 #include <pthread.h>
 #endif
 #include "Guards.h"
+
+#include <boost/core/null_deleter.hpp>
+#include <boost/log/attributes/clock.hpp>
+#include <boost/log/attributes/function.hpp>
+#include <boost/log/expressions.hpp>
+#include <boost/log/sinks/async_frontend.hpp>
+#include <boost/log/sinks/text_ostream_backend.hpp>
+#include <boost/log/sources/global_logger_storage.hpp>
+#include <boost/log/sources/severity_channel_logger.hpp>
+#include <boost/log/support/date_time.hpp>
+
 using namespace std;
-using namespace dev;
 
 //⊳⊲◀▶■▣▢□▷◁▧▨▩▲◆◉◈◇◎●◍◌○◼☑☒☎☢☣☰☀♽♥♠✩✭❓✔✓✖✕✘✓✔✅⚒⚡⦸⬌∅⁕«««»»»⚙
 
+namespace dev
+{
 // Logging
-int dev::g_logVerbosity = 5;
+int g_logVerbosity = 5;
 mutex x_logOverride;
 
 /// Map of Log Channel types to bool, false forces the channel to be disabled, true forces it to be enabled.
 /// If a channel has no entry, then it will output as long as its verbosity (LogChannel::verbosity) is less than
 /// or equal to the currently output verbosity (g_logVerbosity).
 static map<type_info const*, bool> s_logOverride;
 
-bool dev::isChannelVisible(std::type_info const* _ch, bool _default)
+bool isChannelVisible(std::type_info const* _ch, bool _default)
 {
     Guard l(x_logOverride);
```

Comment:
We had hard times in ethminer to figure out what this is about.

Reply:
That's about the feature to temporarily disable output from some channel.
This disabling was used in `Block::enact()` (maybe nowhere else), I removed it for now, don't see any difference so far.